templates/doublezero/index.ts

```diff
@@ -0,0 +1,26 @@
+import {
+	generateHash,
+	generateRandomDomain,
+	type Template,
+	type Schema,
+} from "../utils";
+
+export function generate(schema: Schema): Template {
+	const mainServiceHash = generateHash(schema.projectName);
+	const randomDomain = generateRandomDomain(schema);
+	const envs = [
+		`DOUBLEZERO_HOST=${randomDomain}`,
+		"DOUBLEZERO_PORT=4000",
+		`HASH=${mainServiceHash}`,
+		"SECRET_KEY_BASE=",
```

Comment:
Looking at the documentation it needs a base64, so please use the `generateBase64` function